ci/setup_env.sh

```diff
@@ -43,12 +47,37 @@ function check_signed_tag() {
   fi
 }
 
+function  check_versions_match () {
+  local versions_to_check=("$@")
+
+  if [ "${#versions_to_check[@]}" -eq 1 ]; then
+    echo "Warning: ${FUNCNAME[0]} requires more than one version to be able to compare with.  The build is not going to be released ..."
+    export IS_A_RELEASE="false" && return
+  fi
+
+  versions_match="true"
+  for (( i=0; i<((${#versions_to_check[@]}-1)); i++ )); do
+    [ "${versions_to_check[$i]}" != "${versions_to_check[(($i+1))]}" ] &&
+    { echo -e "Warning: one or more module(s) versions do NOT match. The build is not going to be released ... !!!\nThe versions are ${versions_to_check[*]}"; versions_match="false" && break; }
+  done
+
+  if [ "${versions_match}" == "false" ]; then
+    echo "Versions don't match"
+    export IS_A_RELEASE="false"
+    return
+  fi
```

Comment:
Adding this extra block you are going to print message about not matching version twice. I would leave it as it was before and change `break` on line N61 to `return`